docs/user/03-traces.md

```diff
@@ -502,3 +502,21 @@ System-related spans reported by Istio are filtered out without the opt-out opti
           - name: "kyma-traces"
           randomSamplingPercentage: 60
     ```
+
+### Buffer Filling Up
+
+  Cause: The backend sizing is inadequate
```

Comment:
The word "sizing" might be a bit misleading here. Usually the problem is the possible ingestion rate, not storage capacity.